inc/blocks/namespace.php

```diff
@@ -691,3 +700,99 @@ function get_views( string $block_id, $args = [] ) {
 
 	return $data;
 }
+
+/**
+ * Extract inner blocks from attributes.
+ *
+ * @param array $block The block configuration.
+ * @return array
+ */
+function ssr_inner_blocks_shim( array $block ) : array {
+	if ( $block['blockName'] !== 'altis/shim' ) {
+		return $block;
+	}
+
+	// Populate inner blocks by parsing content.
+	$block['innerBlocks'] = parse_blocks( $block['attrs']['content'] ?? [] );
+
+	// Populate inner content with an array of null placeholders so inner blocks are processed.
+	$block['innerContent'] = array_fill( 0, count( $block['innerBlocks'] ), null );
+
+	return $block;
+}
+
+/**
+ * Registers an inner blocks SSR shim.
+ *
+ * @return void
```

Comment:
If there's no return value, do we need to document it? (I generally omit `@return` if there's no actual return, and save `@return void` for when there _is_ a return, but no actual _data_ is returned.)

Reply:
It’s optional but doesn’t matter too much. I think we’re started to do it more and making things explicit is generally good